### PR TITLE
Add simple 404 page

### PR DESCRIPTION
**Pull request by pydanny**
Test this page by going to a URL that doesn't exist on the page such as "/blarg" or "/404".

### CODE REVIEW

**Comment by ozer619** [comment]
Hi @pydanny thank you so much for your contribution !! The implementation is really a great!! start 

The content looks really great and fonts are perfect too

I noticed that the 404 template appears a bit compressed :
Screenshot_20210829_222455

Take a look at the code for the thank you page as an example of a page that uses a simpler layout .
Perhaps you can use same layout component . experiment with that and let us know if u need any help updating this PR.
You can find us in the chat https://app.element.io/#/room/#mhf:matrix.org ask us if u need further guidance.

File: pages/404.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import Card, { CardParagraph, CardTitle } from '../components/Card';
+import PrimaryLayout from '../components/PrimaryLayout';
+
+export default function NotFound() {
+  return (
+    <>
+      <PrimaryLayout />
+      <Card backgroundImageSource="/images/Volunteer.jpg">
+        <CardTitle>{'404 Page Not Found'}</CardTitle>
+        <CardParagraph>{'Always act in kindness'}</CardParagraph>
```
Comment (ozer619) [inline pages/404.js:12]:
```suggestion
        <CardParagraph>{' The Page you requested was not found.  Always act in kindness by reporting this at https://github.com/margaritahumanitarian/helpafamily/issues/new/choose'}</CardParagraph>
```

**Review by ozer619 — CHANGES_REQUESTED** [review]
I have suggested a change to CardParagraph to make the 404 page more descriptive.

File: pages/404.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+export default function NotFound() {
+  return (
+    <>
+      <PrimaryLayout />
+      <Card backgroundImageSource="/images/Volunteer.jpg">
+        <CardTitle>{'404 Page Not Found'}</CardTitle>
+        <CardParagraph>
+          {
+            ' The page you requested was not found. Always act in kindness by reporting errors like this at https://github.com/margaritahumanitarian/helpafamily/issues/new/choose'
```
Comment (audreyfeldroy) [inline pages/404.js:14]:
```suggestion
            'The page you requested was not found. Always act in kindness by reporting errors like this at https://github.com/margaritahumanitarian/helpafamily/issues/new/choose'
```

Comment (audreyfeldroy) [reply pages/404.js:14]: Take a look at https://nextjs.org/docs/api-reference/next/link and see if you can implement a hyperlink using the `Link` component.

@ozer619 would you like to "Insert a suggestion" via the GitHub PR review tool that does this?

**Review by audreyfeldroy — CHANGES_REQUESTED** [review]
@ozer619 I've left you a practice task to get you familiar with the GitHub PR review tools for suggesting changes.

File: pages/404.js (hunk 2 above)
Comment (ozer619) [inline pages/404.js:14]:
```suggestion
 'The page you requested was not found. Always act in kindness by reporting errors like this at'}
<Link href="https://github.com/margaritahumanitarian/helpafamily/issues/new/choose"
{"here"}
</Link>           
```

**Review by ozer619 — CHANGES_REQUESTED** [review]

**Comment by RedFox0x20** [comment]
Replaced by #183

**Pull request closed without merging**